sdk/openai/Azure.AI.OpenAI/src/Custom/AzureOpenAIModelFactory.cs

```diff
@@ -7,29 +7,192 @@
+            ContentFilterSeverity severity = default,
+            bool isFiltered = default)
+        {
+            return new ContentFilterResult(severity, isFiltered);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of ContentFilterResults for tests and mocking.
+        /// </summary>
+        /// <param name="sexualFilterResult"> The content filter result for the 'sexual' category </param>
+        /// <param name="violenceFilterResult"> The content filter result for the 'violence' category </param>
+        /// <param name="hateFilterResult"> The content filter result for the 'hate' category </param>
+        /// <param name="selfHarmFilterResult"> The content filter result for the 'selfHarm' category </param>
+        /// <returns> A new instance of ContentFilterResults </returns>
+        public static ContentFilterResults ContentFilterResults(
+            ContentFilterResult sexualFilterResult = null,
+            ContentFilterResult violenceFilterResult = null,
+            ContentFilterResult hateFilterResult = null,
+            ContentFilterResult selfHarmFilterResult = null)
+        {
+            return new ContentFilterResults(sexualFilterResult, violenceFilterResult, hateFilterResult, selfHarmFilterResult);
+        }
+        /// <summary>
+        /// Initializes a new instance of PromptFilterResult for tests and mocking.
+        /// </summary>
+        /// <param name="index"> The index of the prompt to associate this filter result with </param>
+        /// <param name="contentFilterResults"> The content filter results associated with the matching prompt index </param>
+        /// <returns> A new instance of PromptFilterResult </returns>
+        public static PromptFilterResult PromptFilterResult(
+            int index = default,
+            ContentFilterResults contentFilterResults = null)
+        {
+            return new PromptFilterResult(index, contentFilterResults);
+        }
+
         /// <summary> Initializes a new instance of Choice. </summary>
         /// <param name="text"> The generated text for a given completions prompt. </param>
         /// <param name="index"> The ordered index associated with this completions choice. </param>
+        /// <param name="contentFilterResults"> The filter category results associated with this completions choice </param>
         /// <param name="logProbabilityModel"> The log probabilities model for tokens associated with this completions choice. </param>
         /// <param name="finishReason"> Reason for finishing. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="text"/> is null. </exception>
         /// <returns> A new <see cref="OpenAI.Choice"/> instance for mocking. </returns>
-        public static Choice Choice(string text = null, int index = default, CompletionsLogProbabilityModel logProbabilityModel = null, CompletionsFinishReason finishReason = default)
+        public static Choice Choice(
```

Comment:
This one (Choice) is also already present in the auto-generated one.

Reply:
All part of that naming difference issue -- will answer on the converged comment (glad we caught this!)